src/figuresClasses.ts

```diff
@@ -1,19 +1,86 @@
-export interface Figure {
+enum Shapes {
+  triangle = 'triangle',
+  circle = 'circle',
+  rectangle = 'rectangle'
```

Comment:
Better name your enums values starting with capital letter
```suggestion
  Triangle = 'triangle',
  Circle = 'circle',
  Rectangle = 'rectangle'
```